CHANGELOG.md

```diff
@@ -38,6 +38,7 @@
 * [BUGFIX] Distributor: Fix a memory leak in distributor due to the cluster label. #4739
 * [ENHANCEMENT] Compactor: uploading blocks no compaction marks to the global location and introduce a new metric #4729
   * `cortex_bucket_blocks_marked_for_no_compaction_count`: Total number of blocks marked for no compaction in the bucket.
+* [FEATURE] Querier: Use streaming for ingester metdata APIs. #4725
```

Comment:
Can you please put this line close together with other `[FEATURE]`?  This helps eyeballing the changes.

Reply:
Done.